docs/editors-guide/project-management.md

```diff
@@ -0,0 +1,76 @@
+**Technical Advisor** | Provide advise and oversight to technical team. | [Chris Mungall](http://biosciences.lbl.gov/profiles/chris-mungall/)
+**Strategic Advisor** | Provide strategic advise and oversight to technical team. | [Melissa Haendel](https://orcid.org/0000-0001-9114-8737) 
+**Stakeholder Engagement/Outreach** | Liase with community users and stakeholders and provide outreach for Mondo resource | [Nicole Vasilevsky](https://orcid.org/0000-0001-5208-3432), [Sabrina  Toro](https://orcid.org/0000-0002-4142-7153), [Sarah Gehrke](https://orcid.org/0000-0003-3245-2880), [Moni Munoz-Torres](https://orcid.org/0000-0001-8430-6039)   | [workshop](https://github.com/monarch-initiative/mondo/issues?q=is%3Aopen+is%3Aissue+label%3Aworkshop)
+
+## Recurring meetings
+Mondo meetings should be added to the Monarch shared google calendar. Members of the curation and technical teams are outlined [here](https://mondo.monarchinitiative.org/pages/contributors/).
+
+- **Curation call**: Thursdays, 9am PT
+- **Technical call**: Fridays, 10am PT
+- **Quarterly Medical Expert calls**: Next meeting is Thursday, Dec 01, 2022, 12-1pm PT. Future calls will be scheduled.
+
+## Mondo workshops
+- Any upcoming workshops should be organized and advertised.
+
+### Organizing workshops
+
+Ideally we should have community based Mondo workshops every 4-6 months.
+
+### Advertising workshops
+
+  - on our [website](https://mondo.monarchinitiative.org/pages/workshop/)
+  - on our Slack channel
+  - on the mailing list
+
+## Set milestones and triage tickets
+- All [tickets](https://github.com/monarch-initiative/mondo/issues) should be:
+  1. Acknowledged, thank the contributor for the ticket 
+  2. Assigned to a curation or technical team member
+  3. Assigned to a monthly milestone
+  4. Given a label
+
+## Mondo Releases
+
+- Please refer to the [pre-release checklist](https://mondo.readthedocs.io/en/latest/editors-guide/pre-release-checklist/) for necessary tasks before the release
+- Mondo should be released on a monthly basis on/around the first of the month. Instructions are [here](https://mondo.readthedocs.io/en/latest/developer-guide/release/).
+- If there are any delays with the release (more than a couple days), please communicate these to the Mondo users on the mailing list and in Slack. Some of our users depend on the regular releases, such as EFO
+
+## Grant reporting
+
+- Link Mondo tickets to [Phenomics First Operations Board](https://github.com/monarch-initiative/phenomics_first_resource/issues?q=is%3Aissue+is%3Aopen+label%3A%22Aim+2+-+MONDO%22)
+- Ensure deliverables for P1 grant are being addressed
+- Coordinate reporting for yearly RPPR
+
```

Comment:
I would like to kindly suggest adding one bullet point about sending all dissemination artifacts to the 'nuggets' address for Monarch/TISLab (to help RPPR prepping). It could just be appended to that line item. Thanks! :bowtie: